src/interface/interface.cpp

```diff
@@ -145,7 +145,7 @@ bool loadInterfaceResources()
 	status_bmp = loadImage("images/system/StatusBar.png");
 	character_bmp = loadImage("images/system/CharacterSheet.png");
 	hunger_bmp = loadImage("images/system/Hunger.png");
-	minotaur_bmp = loadImage("images/system/minotaur.png");
+	minotaur_bmp = loadImage("images/sprites/minotaur.png"); // the file "images/system/minotaur.png" doesn't exist in current Data
```

Comment:
Yes, I needed the game to work, I haven't found that ressoyrce on the GoG data nor on the git source.
I can make a pull request to put it back to the correct value.